plugins/accuracy.cc

```diff
@@ -38,6 +39,8 @@ using namespace std;
 namespace Datacratic {
 namespace MLDB {
 
+typedef std::vector<std::pair<RowName, std::vector<std::tuple<ColumnName, CellValue, Date> > > > Rows;
```

Comment:
We have some many different types that contain _Row_ that it is becoming confusing!  There are no action required for this PR but at one point we will want to standardize and centralize these.